docs/README.rst

```diff
@@ -252,6 +254,28 @@ Some of these cert/key pairs can be wildcard (\*.mycf.com).
 
 .. _health checks:
 
+JSESSIONID Session Persistence
+------------------------------
+
+The |cfctlr| supports session persistence on the BIG-IP through cookies. Enabled by default on the |cfctlr|. To disable session persistence define the ``session_persistence`` setting as follows:
+
+.. code-block:: yaml
+
+   session_persistence: false
+
+To configure session persistence for the |cfctlr| your application must set cookies in its HTTP response headers as follows:
+
+``Set-Cookie: JSESSIONID=<value>; Max-Age=<age>``
+
+.. note::
+   - The cookie name of JSESSIONID is case insensitive so jsessionid and JSessionID are also valid.
+   - The cookie value can be anything as long as it is distinguishable from other cookies set by other responses.
+   - Setting the cookie Max-Age to the following values will result in the following behavior:
```

Comment:
change to a simple table:
```
.. table:: Cookie Max-Age values

   ==== =====================================================  
   Age    Behavior
   ==== =====================================================  
   < 0     Session persists until client connection closes or one hour elapses.
   > 0     Session persists until the Max-Age of the cookie.
   = 0     Delete session persistence record if one exists on the BIG-IP system.
   ==== =====================================================  
```

(The formatting is off in the displayed version, must be a markdown thing. Click Edit to get the correct version to copy and paste.)

Reply:
Thanks Jodie I will make these changes.